File: plsql/Java/PlSqlLexerBase.java
package PlSqlParseTree;

import org.antlr.v4.runtime.*;

public abstract class PlSqlLexerBase extends Lexer
{
    public PlSqlLexerBase(CharStream input)
	{
		super(input);
    }

    protected boolean IsNewlineAtPos(int pos)
    {
        int la = _input.LA(pos);
        return la == -1 || la == '\n';
    }
}

File: plsql/CSharp/PlSqlLexerBase.cs
using Antlr4.Runtime;

namespace PlSqlParseTree
{
    public abstract class PlSqlLexerBase : Lexer
    {
        public PlSqlLexerBase(ICharStream input)
            : base(input)
        {
        }

        protected bool IsNewlineAtPos(int pos)
        {
            int la = _input.La(pos);
            return la == -1 || la == '\n';
        }
    }
}